src/utils/env.ts

```diff
@@ -4,6 +4,7 @@ module.exports = (() => {
     require('child_process')
       .execSync('git rev-parse --sq HEAD')
       .toString()
+      .replace(/^'/, '')
```

Comment:
Without this, we were getting a leading `'` in the `GIT_SHA` when building for OSS with `yarn build`. I don't _think_ this is used anywhere else (that I could find)...but I made it a regex replacement so that it doesn't just drop the first character - it only cuts off a leading `'`.